### PR TITLE
Adding Ubuntu x64 binary for Ruby 2.4.0 (#3867)

### LINKED ISSUE #3867: Add Ubuntu x64 binary for Ruby 2.4.0

### PR DESCRIPTION
**Pull request by raelgc**
Fixes #3867

### CODE REVIEW

**Comment by raelgc** [comment]
@pkuczynski Just added! Sorry!

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,12 +8,13 @@
...
 * Add support for KDE neon [\#3828](https://github.com/rvm/rvm/pull/3828)
-* Allow to remove undesired libraries breaking the ruby build [\#3851](https://github.com/rvm/rvm/issues/3851) 
+* Allow to remove undesired libraries breaking the ruby build [\#3851](https://github.com/rvm/rvm/issues/3851)
+* Ubuntu x64 binary for ruby 2.4.0 [\#3867](https://github.com/rvm/rvm/issues/3867)
 
 #### Bug fixes:
 * $PATH become empty after __rvm_unload executed [\#3847](https://github.com/rvm/rvm/pull/3847)
 * RVM incorrectly tries to install llvm 3.5 when trying to install Rubinius 3 [\#3848](https://github.com/rvm/rvm/pull/3848)
-* Failing openssl.patch for Ruby 1.9.3 [\#3831](https://github.com/rvm/rvm/issues/3831) 
+* Failing openssl.patch for Ruby 1.9.3 [\#3831](https://github.com/rvm/rvm/issues/3831)
```
Comment (pkuczynski) [inline CHANGELOG.md]: Why this change?

File: CHANGELOG.md [hunk 2]
```diff
@@ -62,7 +63,7 @@
 
 #### Binaries:
 * Ruby 2.3.0, 2.3.1 [\#3774](https://github.com/rvm/rvm/issues/3774), 2.3.3 [\#3822](https://github.com/rvm/rvm/issues/3822) binaries for Debian 8 x86_64
-* Ruby 2.0.0-p648, 2.1.5, 2.1.9, 2.2.5, 2.3.0 [\#3693](https://github.com/rvm/rvm/issues/3693), 2.2.6 and 2.3.2 [\#3809](https://github.com/rvm/rvm/issues/3809), 2.3.3 [\#3820](https://github.com/rvm/rvm/issues/3820) binaries for Ubuntu 16.04
```
Comment (pkuczynski) [inline CHANGELOG.md]: Why this change?
